test/e2e-tests.sh

```diff
@@ -178,7 +182,11 @@ test_dashboard() {
 
   # In case of failure we'll upload videos of the failing tests
   # Our Cypress config will delete videos of passing tests before exiting
-  docker run --rm --network=host $CYPRESS_ENV -v $VIDEO_PATH:/home/node/cypress/videos dashboard-e2e || fail_test "Browser E2E tests failed"
+  CYPRESS_SPEC=""
+  if [[ "$readonly" == true ]]; then
+    CYPRESS_SPEC="-- --spec ""cypress/e2e/common/*.js"""
```

Comment:
```suggestion
    CYPRESS_SPEC='-- --spec "cypress/e2e/common/**/*"'
```

gives us a bit more flexibility in how we organise tests in future, and Cypress is already configured to look for the `.cy.js` extension so we don't need to repeat it here.